azure-kusto-ingest/azure/kusto/ingest/_resource_manager.py

```diff
@@ -165,3 +173,41 @@ def retrieve_service_type(self):
 
     def set_proxy(self, proxy_url: str):
         self._kusto_client.set_proxy(proxy_url)
+
+    def create_resources_by_storage_account(self, resources):
+        resources_by_storage_account = dict()
+        for resource in resources:
+            if resource.storage_account_name not in resources_by_storage_account:
+                resources_by_storage_account[resource.storage_account_name] = list()
+            resources_by_storage_account[resource.storage_account_name].append(resource)
+
+        return resources_by_storage_account
+
+    def select_with_round_robin(self, original_dict):
```

Comment:
Different name here, since it not only selects with round robin, but also shuffles both accounts in containers in accounts.
`shuffled_with_round_robin` maybe